Security.sln

```diff
@@ -79,6 +79,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.Authen
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "WsFedSample", "samples\WsFedSample\WsFedSample.csproj", "{5EC2E398-E46A-430D-8E4B-E91C8FC3E800}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.Authorization.Middleware", "src\Microsoft.AspNetCore.Authorization.Middleware\Microsoft.AspNetCore.Authorization.Middleware.csproj", "{E2D9B067-CD74-4EA8-9CBA-D303DB63C1AC}"
```

Comment:
Do we need a project for this? Is the reason that we don't have a project that ties together HTTP and AuthZ?

Reply:
Authorization.Policy has both